onos-classic/templates/statefulset.yaml

```diff
@@ -118,7 +122,7 @@ spec:
               - /root/onos/bin/check-onos-status
           initialDelaySeconds: 300
           periodSeconds: 15
-          timeoutSeconds: 5
+          timeoutSeconds: {{ mul .Values.probe_timeout 2 }}
```

Comment:
Peronsal thoughtcrimes, I think double the timeout is a bit too much, maybe 1.2 or 1.5 ?

Reply:
it seems I cannot do this math: https://github.com/Masterminds/sprig/blob/bf29da0d74f74aeb5c3e3e7207eab76c28ac4049/functions.go#L182

Reply:
seems stupid to convert to int64 but nothing we can do, thanks